class/SyncHandlers/SyncHandlerBase.php

```diff
@@ -39,6 +39,11 @@ public static function sync_many(array $shortcodes)
     }
   }
 
+  public static function sync_one(string $shortcode)
+  {
+    static::sync_many(array($shortcode));
```

Comment:
We should use the cleaner `[]` syntax for arrays instead of `array()`

```suggestion
    static::sync_many([$shortcode]);
```